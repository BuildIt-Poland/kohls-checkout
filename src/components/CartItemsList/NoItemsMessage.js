import styled from 'styled-components';

import { spacingLarge } from '../../styles/designTokens';
import Text from '../Text';

const NoItemsMessage = styled(Text)`
  padding-bottom: ${spacingLarge};
`;

export default NoItemsMessage;
