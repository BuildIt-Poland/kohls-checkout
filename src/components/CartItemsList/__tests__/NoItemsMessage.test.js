import React from 'react';
import { create } from 'react-test-renderer';

import NoItemsMessage from '../NoItemsMessage';

describe('COMPONENT - CartItemsList NoItemsMessage', () => {
  it('renders ItemsList component', () => {
    const component = create(<NoItemsMessage />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
