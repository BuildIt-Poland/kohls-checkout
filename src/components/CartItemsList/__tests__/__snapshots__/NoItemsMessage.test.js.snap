// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - CartItemsList NoItemsMessage renders ItemsList component 1`] = `
.c0 {
  font-size: 1.8rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-weight: 400;
  line-height: 1.4;
  margin: 0;
  margin-bottom: 1rem;
  color: #222222;
  color: #222222;
  padding-bottom: 3rem;
}

.c0:last-child {
  margin-bottom: 0;
}

<p
  className="c0"
/>
`;
